django-backend/fecfiler/f3x_summaries/views.py

```diff
@@ -13,6 +15,36 @@ class F3XSummaryViewSet(CommitteeOwnedViewSet):
     in CommitteeOwnedViewSet's implementation of get_queryset()
     """
 
-    queryset = F3XSummary.objects.all().order_by("-id")
+    queryset = F3XSummary.objects.alias(report_code_label=Case(
```

Comment:
along with what i suggested in the app PR, could this mapping live in a data structure that lives here and is passable to the front end?

Reply:
If the data were dynamic and changed with some frequency I would want the labels to be pulled from one place. However, considering the stability of the labels and the rarity in which they will change (if ever) I think it better to forgo an HTTP call and maintain 2 separate lists.